src/coreclr/debug/inc/runtimeinfo.h

```diff
@@ -14,10 +15,11 @@ typedef unsigned char SYMBOL_INDEX;
 typedef struct _RuntimeInfo
 {
     const char Signature[18];
-    int Version;
+    const int Version;
     const SYMBOL_INDEX RuntimeModuleIndex[24];
```

Comment:
Do we have a use case for RuntimeModuleIndex? Or is this for completeness? You should be able to get all the debugging artifacts from the other two? Or is this needed for the metrics?

Reply:
We (clrmd) uses it for the cross-DAC case.  We don't have an identity key for the Linux cross-DAC windows dll only the Linux .so files.

dbgshim doesn't use it.